pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -1227,6 +1242,22 @@ func doActivate(ctx *domainContext, config types.DomainConfig,
 		return
 	}
 
+	//TODO: mountPoints must be fill with target paths on the container once we decide UI to domainmgr flow for vol mount.
+	mountPoints := make([]MountPoints, 0)
+	mpFileName := mountPointFileName(config.AppNum)
+	mpFile, err := os.Create(mpFileName)
+	if err != nil {
+		log.Fatal("os.Create for ", mpFileName, err)
```

Comment:
Instead of Fatal - better treat it as an error and take the App Instance to Error state.
Fatal here would likely result in a reboot loop.

Reply:
Changed it to Error